framework/web/View.php

```diff
@@ -504,7 +505,7 @@ private function registerFile($type, $url, $options = [], $key = null)
                 'class' => AssetBundle::className(),
                 'baseUrl' => '',
                 'basePath' => '@webroot',
-                (string)$type => [!Url::isRelative($url) ? $url : ltrim($url, '/')],
+                (string)$type => [ArrayHelper::merge([!Url::isRelative($url) ? $url : ltrim($url, '/')], $originalOptions)],
```

Comment:
Should it be wrapped in array?

Reply:
What I mean is that the output will be `[[...]]` instead of `[...]` - is it the way you want it? Also while you are at it, could you correct the old `$asssetManagerAppendTimestamp` name (few lines above)? It should only have 2 `s` ;)